srcpkgs/antimicrox/template

```diff
@@ -12,23 +12,7 @@ maintainer="fosslinux <fosslinux@aussies.space>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/AntiMicroX/antimicrox"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=53c6b0e6c8ea8c3a5527b56e1ad052eae983ba36abfc090769c6164bc5f342dd
-
-libantimicrox_package() {
```

Comment:
why remove these subpackages?

Reply:
the cmake file no longer produces them

Reply:
ok, then the symlinks need to be removed, and those subpackages need to be added to the `replaces` list in `removed-packages`

Reply:
and also the shlib needs to be removed from `common/shlibs`

Reply:
still need to `rm srcpkgs/libantimicrox srcpkgs/libantimicrox-devel` and add them to the commit